Dockerfile

```
@@ -0,0 +1,37 @@
+# Build stage
+FROM ghcr.io/userver-framework/ubuntu-22.04-userver-pg-dev:latest AS builder
```

Hadolint (2.14.0)

[warning] Using latest is prone to errors if the image will ever update. Pin the version explicitly to a release tag

(DL3007)